src/lib/types.ts

```diff
@@ -970,3 +970,8 @@ export type CallRefreshTokenResult =
       session: null
       error: AuthError
     }
+
+export type PageParams = {
+  page?: string,
+  per_page?: string
```

Comment:
```suggestion
  page?: number,
  perPage?: number
```

Reply:
nit: preference of camelCase over snake case here

Reply:
using the `number` as the type is better for the DX since it prevents devs from using non-numbers as values

Reply:
also, gotrue handles the type conversion for us